src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.dsde.rawls.dataaccess.slick
+
+import java.sql.Timestamp
+import java.util.UUID
+
+/**
+ * Created by dvoet on 2/4/16.
+ */
+case class AttributeRecord(id: Long,
+                           name: String,
+                           valueString: Option[String],
+                           valueNumber: Option[Double],
+                           valueDate: Option[Timestamp],
+                           valueBoolean: Option[Boolean],
+                           listIndex: Option[Int])
+
+trait AttributeComponent {
+  this: DriverComponent =>
+
+  import driver.api._
+
+  class AttributeTable(tag: Tag) extends Table[AttributeRecord](tag, "ATTRIBUTE") {
+    def id = column[Long]("id", O.PrimaryKey, O.AutoInc)
+    def name = column[String]("name")
+    def valueString = column[Option[String]]("value_string")
```

Comment:
What about the idea we had of a "value type" column, so you knew this was a String attribute without checking for Option/null?